Orders/Client/EP1/co01010001.csv

```diff
@@ -0,0 +1,3 @@
+Title#0,"""Level Cap Trial II """
```

Comment:
Please do not CHANGE this text or add spaces to bypass the EN string check

Reply:
Okay, but how can I avoid CircleCI's error in this case?